packages/client/tests/functional/issues/11789-sqlite-timed-out/_matrix.ts

```diff
@@ -0,0 +1,4 @@
+import { defineMatrix } from '../../_utils/defineMatrix'
+import { allProviders } from '../../_utils/providers'
+
+export default defineMatrix(() => [allProviders])
```

Comment:
```suggestion
import { defineMatrix } from '../../_utils/defineMatrix'
import { Providers } from '../../_utils/providers'

export default defineMatrix(() => [
  [
    {
      provider: Providers.SQLITE,
    },
  ],
])
```
Would be best I guess

Reply:
Or rename the directory?

Reply:
Alright, I've renamed the directory.